dashboard-api/Dockerfile.test

```diff
@@ -0,0 +1,9 @@
+FROM golang:alpine
+
+RUN apk add --no-cache \
+    gcc \
+    libc-dev \
+    readline-dev
+
+WORKDIR /github.com/vlab-research/vlab/dashboard-api
+RUN go get -u github.com/nathany/looper
```

Comment:
Why do we need a `Dockerfile.test`? Couldn't we just reuse `Dockerfile.dev`?

The idea of having an additional dockerfile apart from `Dockerfile`, is that we wanted to have a container for _development_ purposes, and leave `Dockerfile` with only the strictly necessary dependencies. 

The `Dockerfile.dev` will then contain everything required for local development and testing, as this dockerfile doesn't need to be _optimize_ as `Dockerfile` would.

Reply:
In this specific case, it looks like both `looper` and `air` are essentially doing the same thing, right?

Reply:
Yes I will group them into a single Dockerfile.dev 👌